pkg/asset/installconfig/aws/validation.go

```diff
@@ -151,17 +153,19 @@ func validateAMI(ctx context.Context, config *types.InstallConfig) field.ErrorLi
 			computesHaveAMISpecified = false
 		}
 	}
+
 	if controlPlaneHasAMISpecified && computesHaveAMISpecified {
 		return nil
 	}
 
 	// accept AMI that can be copied from us-east-1 if the region is in the standard AWS partition
-	if partition, partitionFound := endpoints.PartitionForRegion(endpoints.DefaultPartitions(), config.Platform.AWS.Region); partitionFound {
-		if partition.ID() == endpoints.AwsPartitionID {
-			return nil
-		}
+	regions := allKnownRegions()
+	logrus.Warnf("region: %s", config.Platform.AWS.Region)
+	logrus.Warnf("regions: %+v", regions)
+	if sets.New(regions...).Has(config.Platform.AWS.Region) {
+		return nil
 	}
-
+	os.Exit(1)
```

Comment:
Is this leftover from some debugging/testing?

Reply:
It is :) thought I removed all of them. I did multiple passes oops!